index.js

```diff
@@ -30,6 +30,7 @@ const NoFormData = createError('FST_NO_FORM_DATA', 'FormData is not available',
 function setMultipart (req, payload, done) {
   req[kMultipart] = true
   req.rawBody = payload.rawBody
+  fastify.decorateRequest('rawBody', req.rawBody)
```

Comment:
you decorate the request when you register the plugin. then yo can assign rawbody to the fastify request.